projects/management-software-sync/backend/samples/schema/lists.json

```diff
@@ -1,8 +1,8 @@
 [
-    {
-        "id_prodotto": "",
-        "listino": "",
-        "netto": "",
-        "umd": ""
-      }
-]
\ No newline at end of file
+  {
+    "id_prodotto": "",
+    "listino": "",
+    "netto": "",
+    "umd": ""
```

Comment:
Not clear too.

Reply:
If you refer to "umd" , it can be omitted